test/preset-icons.test.ts

```diff
@@ -62,4 +62,8 @@ describe('preset-icons', () => {
     expect(css).toContain('data:image/svg+xml;utf8,%3Csvg')
     await expect(css).toMatchFileSnapshot('./assets/output/preset-icons-unit-svg-prologue.css')
   })
+
+  it('exports default collection list', () => {
+    expect(icons).toMatchSnapshot()
```

Comment:
We probably don't need to test it. The list is kinda external, whenever the list updates, it introduce uncessary snapshot changes.